rubin_sim/maf/plots/spatial_plotters.py

```diff
@@ -177,6 +178,7 @@ def __call__(self, metric_value_in, slicer, user_plot_dict, fignum=None):
             )
             mask = np.zeros(metric_value.size)
             mask[np.where(metric_value == slicer.badval)] = 1
+            # XXX--hm, this looks broken
```

Comment:
I'm not sure what you mean here - could you clarify?

Reply:
Below it was using something that wasn't imported. I probably fixed it, but this section still has no unit test coverage.